package.json

```diff
@@ -6,7 +6,12 @@
     "start": "ng serve",
     "build": "ng build",
     "watch": "ng build --watch --configuration development",
-    "test": "ng test"
+    "test": "ng test",
+    "prepare": "husky",
+    "lint": "eslint . && stylelint src/styles/**/*.scss",
```

Comment:
It's better to set a general path for all scss files like this \"src/**/*.{css,scss}\"

You also can run the npm scripts you've created: "lint":  "npm run eslint && npm run stylelint"

Reply:
Done